### ISSUE TITLE
Don't see number of commits when you're seeing commits or files changed of a Pull Request

### ISSUE THREAD

**Issue by xxxtonixxx** [gogits/gogs#2251]
- Normal view of repository:
![image](https://cloud.githubusercontent.com/assets/7110786/11916074/0ae72080-a6c6-11e5-820d-1d740fb16d1a.png)

- Bug:
![bug](https://cloud.githubusercontent.com/assets/7110786/11916066/da729060-a6c5-11e5-85cd-ff239e949d1c.png)

Gogs' version: 0.8.10

**Comment by xxxtonixxx** [comment]
Hmm, It seems that the template that is loaded it's wrong...

![image](https://cloud.githubusercontent.com/assets/7110786/11916085/bbceace2-a6c6-11e5-9887-38029981bb0e.png)

**Comment by Unknwon** [comment]
Please try `develop` again!

**Comment by xxxtonixxx** [comment]
Thank you!
I can't try develop branch... I will wait to try when it's in try.gogs.io :+1:

- It show "New issue" when it must see "New Pull Request"

![image](https://cloud.githubusercontent.com/assets/7110786/11916129/9df755ae-a6c9-11e5-9777-4eb6b4bcaacb.png)

**Comment by Unknwon** [comment]
1. Demo site is updated.
2. 

> It show "New issue" when it must see "New Pull Request"

Sorry can you rephrase this sentence?

**Comment by xxxtonixxx** [comment]
Sorry! :sweat:

Hmm

- Thant's a Pull Request and there is a button that it says "New Pull Request"
![image](https://cloud.githubusercontent.com/assets/7110786/11916186/5c1c5852-a6cc-11e5-834b-60f6899f8664.png)

- But, when you change of tab (commits or files changed), the button doesn't say "New Pull Request", it say "New Issue" and that doesn't make sense
![image](https://cloud.githubusercontent.com/assets/7110786/11916190/c0e4feb0-a6cc-11e5-9f6b-67e3cedc70ac.png)

**Comment by Unknwon** [comment]
Updated `develop` branch and demo site.

**Comment by xxxtonixxx** [comment]
It's ok!

Thank you!

**Comment by Unknwon** [comment]
Thanks your confirmation!
